### ISSUE TITLE
Arbitrary JavaScript execution through autolink

### ISSUE THREAD

**Issue by jeffw16** [justinpotts/pattr#29]
If one pastes in
```
https://www.mywikis.com"onclick="javascript:disconnect()"
```
into the chatbar, the client will disconnect on Chrome.

Well, let's just say this is also possible:
```
https://www.mywikis.com"onclick="javascript:alert('Pattr_got_hacked')"
```

![screen shot 2015-11-24 at 9 14 36 pm](https://cloud.githubusercontent.com/assets/11380894/11387456/4b2665fa-92f0-11e5-90c6-f2ee79113ffe.png)

The potential of abuse is very great. I recommend an escalation in severity to critical.

This needs to be fixed immediately by setting cgi.escape's second parameter to True, which will look something like this but I'm not sure since my Python isn't the strongest:
```
message['data'] = escape ( message['data'], quotes=True )
```

**Comment by justinpotts** [comment]
Fixed in d9dc2b6

**Comment by jeffw16** [comment]
![thanksgif](https://cloud.githubusercontent.com/assets/11380894/11390174/fe93e664-930c-11e5-87b6-7d135143b724.gif)
